src/index.html

```diff
@@ -1,12 +1,375 @@
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+            </div>
+            <div class="card__rating-count">Отзывов: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Цена:</p>
+            <p class="card__price-amount">
+              <strong>
+                69 999 грн
+              </strong>
+            </p>
+          </div>
+          <button
+            class="card__btn"
+            data-qa="hover"
+          >
+            Купить
+          </button>
+        </div>
+
+        <div class="card products__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Product image"
+          >
+          <h2 class="card__heading">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+            </div>
+            <div class="card__rating-count">Отзывов: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Цена:</p>
+            <p class="card__price-amount">
+              <strong>
+                69 999 грн
+              </strong>
+            </p>
+          </div>
+          <button
+            class="card__btn"
+            data-qa="hover"
+          >
+            Купить
+          </button>
+        </div>
+
+        <div class="card products__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Product image"
+          >
+          <h2 class="card__heading">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+            </div>
+            <div class="card__rating-count">Отзывов: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Цена:</p>
+            <p class="card__price-amount">
+              <strong>
+                69 999 грн
+              </strong>
+            </p>
+          </div>
+          <button
+            class="card__btn"
+            data-qa="card-hover"
```

Comment:
```suggestion
            data-qa="card-hover"
```
this data-qa should be added to the first card, not to the last